docs/source/installation.rst

```diff
@@ -2,152 +2,133 @@ Installation
+    Currently, ``ivadomed`` supports GPU/CPU on ``Linux`` and ``Windows``, and CPU only on ``macOS`` and `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/>`_.
 
-Open your bash/zsh script file with editor on your computer.
+Step 1: Setup dedicated python environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    If you are using bash shell
-    ::
+    You can setup ``ivadomed`` using either Conda or Venv:
 
-        vim ~/.bashrc
+    .. tabs::
 
-    If you are using zsh shell
-    ::
-        
-        vim ~/.zshrc
+        .. tab:: Install via ``venv``
 
-Write in your .bashrc/.zshrc file with following line.
+            1. Setup Python Venv Virtual Environment.
 
-::
+                ``ivadomed`` requires Python >= 3.6 and <3.10.
 
-    export HDF5_USE_FILE_LOCKING='FALSE'
+                First, make sure that a compatible version of Python 3 is installed on your system by running:
 
-Save this change and restart your terminal to apply the change.
+                .. code::
 
-There are fundamentally two different approaches to install ``IvadoMed``:
+                   python3 --version
 
-1) Install via Conda
-    This is the easiest way for personal computers.
+                If your system's Python is not 3.7, 3.8, or 3.9 (or if you don't have Python 3 installed at all), please `install Python <https://realpython.com/installing-python/>`_ before continuing.
 
-2) Install via Venv/VirtualEnv
-    This is compatible with ComputeCanada cluster environment.
+                Once you have a supported version of Python installed, run the following command (replacing ``3.X`` with the Python version number that you installed):
 
-Approach 1: Conda
-------------------
+                .. code::
 
-Step 1: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+                   python3.X -m venv ivadomed_env
 
-::
+            2. Activate the new virtual environment (default named ``ivadomed_env``)
 
-    git clone https://github.com/neuropoly/ivadomed.git
-    cd ivadomed
+                ::
 
-Step 2: Create new Conda Env called IvadoMedEnv (may take 5 to 15 minutes)
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+                    source ivadomed_env/bin/activate
 
-::
+        .. tab:: Install via ``conda``
 
-    conda env create --file environment.yml
+            1. Create new conda environment using ``environment.yml`` file
 
-Step 3 : Activate environment and use
-++++++++++++++++++++++++++++++++++++++
+                ::
 
-::
+                    conda env create --name ivadomed_env
 
-    conda activate IvadoMedEnv
+            2. Activate the new conda environment (default named ``ivadomed_env``)
```

Comment:
Tiny nitpick: The text "(default named ``ivadomed_env``)" might be a bit unnecessary, since the name ``ivadomed_env`` was chosen in the step immediately before.